index.html

```diff
@@ -77,183 +84,254 @@ <h2>My Recent<br>Works</h2>
+              <div class="card-desktop card-work-6 card">
+                <img src="images/work6.jpg" width="369" height="252" alt="Placeholder Image">
                 <h2 class="text-work">Multi-Post Stories<br>Gain+Glory</h2>
-                <div class="technology-holders">
-                  <div class="technology-holder">Ruby on Rails</div>
-                  <div class="technology-holder">CSS</div>
-                  <div class="technology-holder">JavaScript</div>
-                  <div class="technology-holder">HTML</div>
+                <div class="technology-holders-desktop technology-holders">
+                  <div class="technology-holder-desktop technology-holder">Ruby on Rails</div>
+                  <div class="technology-holder-desktop technology-holder">CSS</div>
+                  <div class="technology-holder-desktop technology-holder">JavaScript</div>
+                  <div class="technology-holder-desktop technology-holder">HTML</div>
                 </div>
-                <div class="button-holder">
-                     <a href="#" class="button">See Project</a>
+                <div class="button-holder-desktop button-holder">
+                     <a href="#" class="button-desktop button">See Project</a>
                 </div>
               </div>
           </div>
     </section>
-    <section class="image">
-        <div></div>
-    </section>
-    <section>
-        <img class="figure1" src="images/figure1.svg" alt="Figure" width="68px" height="127px">
+    <section class="about-container-title">
+    <section class="figure-holder-1-desktop image">
+        <img class="figure1-desktop" src="images/figure1.svg" alt="Figure" width="288px" height="126px">
     </section>
-    <section class="about-frame about">
-        <div>
-            <div class="about-box">
-                <h2>About me</h2>
-            </div>
-        </div>
-    </section>
-    <section class="frame-about">
-        <div>
-            <p class="frame3">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don't hesitate to contact me.
+
+    <section class="about-me-desktop about-frame">
+        <section>
+            
+        </section>
+        <section>
+            
+            <section class="frame3">
+                <section class="figures2-3">
+                    <img class="figure2" src="images/figure2-mobile.png" alt="Figure" width="90px" height="56.29px">
+            <section>
+                <div class="about-box about">
+                    <h2>About me</h2>
+                </div>
+            <p class="textaboutme-desktop ">
+                
+                I can help you build a product , feature or website Look through some of my work and<br>experience! If you like what you see and have a project you need coded,<br>
+                don’t hesitate to contact me.
```

Comment:
- [ ] Geat job you did here 👏 .. However, kindly use the right font and color as specified on the figma design.
- On Figma, the color i `background: #42526E;` and its not bold

## Figma:
![image](https://github.com/AztkMarty/Portfolio-Setup/assets/50990925/95b9e176-e7fc-42ef-a400-aa6d89789966)

## Your work:
![image](https://github.com/AztkMarty/Portfolio-Setup/assets/50990925/d758350b-ae99-457d-812f-061a92751833)